src/features/create-quiz/create-quiz-second-card/create-quiz-second-card.tsx

```diff
@@ -119,39 +170,11 @@ export const CreateQuizSecondCard: React.FC = observer(() => {
   });
 
   const getFormErrorMessage = (name: any) => {
-    return (
-      isFormFieldValid(name, formik) && (
-        <small className="p-error">{formik.errors[name]}</small>
-      )
+    const returnedValue = isFormFieldValid(name, formik) && (
+      <small className="p-error">{formik.errors[name]}</small>
     );
-  };
 
-  const changeCheckedAnswer = (index: number) => {
-    if (formik.values.questionType.name === "Single answer") {
-      setAnswerList((currentState) => {
-        let arrayCopy = [...currentState];
-        arrayCopy = arrayCopy.map((answer, i) => {
-          if (i === index) {
-            return {
-              ...answer,
-              isCorrectAnswer: !answer.isCorrectAnswer,
-            };
-          } else {
-            return {
-              ...answer,
-              isCorrectAnswer: false,
-            };
-          }
-        });
-        return arrayCopy;
-      });
-    } else if (formik.values.questionType.name === "Multiple answers") {
-      setAnswerList((currentState) => {
-        let arrayCopy = [...currentState];
-        arrayCopy[index].isCorrectAnswer = !arrayCopy[index].isCorrectAnswer;
-        return arrayCopy;
-      });
-    }
+    return returnedValue;
```

Comment:
The value could be directly returned without using a new constant 'returnedValue'.